db/schema.rb

```diff
@@ -12,24 +10,44 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_210_323_160_745) do
+ActiveRecord::Schema.define(version: 2021_03_31_054326) do
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
+  enable_extension "plpgsql"
```

Comment:
again, rubocop will mark this as an offense, always make sure, overcommit is not bypassed 
<img width="1314" alt="Screen Shot 2021-04-01 at 10 17 16 AM" src="https://user-images.githubusercontent.com/32864278/113234253-814d1780-92d3-11eb-8f34-a16a4974d406.png">